libvast/vast/expression_visitors.hpp

```diff
@@ -165,6 +165,41 @@ struct event_evaluator {
   relational_operator op_;
 };
 
+/// Evaluates a single row of a table slice over a [resolved](@ref
+/// type_extractor) expression.
```

Comment:
You could add a `@relates` here since the implementation looks very familiar to one that I already know.